src/app/ElectronBootstrap.js

```diff
@@ -30,7 +30,8 @@ module.exports = class ElectronBootstrap {
         ];
         this._directoryMap = {
             'cache': this._configuration.applicationCacheDirectory,
-            'plugins': this._configuration.applicationUserPluginsDirectory
+            'plugins': this._configuration.applicationUserPluginsDirectory,
+            'root': this._configuration.applicationUserDataDirectory
```

Comment:
i was looking for a way to get into the root userData directory to scan for which files were already downloaded.
this seemed like the correct way to do it.
looking for feedback on this

Reply:
Something i forgot while doing this is that it eventually has to work with the `bookmark directory` function. added to the todo list

Reply:
No need for this, the Storage.mjs has access to the path (`electron.remote.app.getPath( 'userData' )`)